src/index.js

```diff
@@ -86,12 +86,14 @@ class LumigoPlugin {
 			}
 		} else if (runtime === "python") {
 			await this.ensureLumigoPythonIsInstalled();
-
+			const { isZip } = await this.getPythonPluginConfiguration();
+			this.verboseLog(`Python plugin zip status ${isZip}`);
 			for (const func of functions) {
 				const handler = await this.createWrappedPythonFunction(
 					func,
 					token,
-					parameters
+					parameters,
+					isZip
```

Comment:
can we move `isZip` into `parameters`? would that make sense?

Reply:
No, the first are parameters that we pass to the tracer itself, isZip is for the way we wrap the code